db/postgres/postgres_types.go

```diff
@@ -202,3 +204,19 @@ type Compatibility struct {
 	State string
 	Note  string
 }
+
+type ModLink struct {
```

Comment:
Please rename to `ModArch`

Reply:
https://github.com/satisfactorymodding/smr-api/pull/12/commits/bdd565d8997568955b8e33bcb352c38c7af5c2c5